packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3657,6 +3657,7 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     super.describeSemanticsConfiguration(config);
     config.isSemanticBoundary = container;
     config.explicitChildNodes = explicitChildNodes;
+    assert((scopesRoute == true && explicitChildNodes == true) || scopesRoute != true);
```

Comment:
The assert should probably include a message explaining that you have to set explicitChildNodes to true when scropesRoute is true.

Reply:
Done